frontend/src/pages/_app.tsx

```diff
@@ -0,0 +1,108 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+import Head from 'next/head';
+import Script from 'next/script'
+import type { AppProps } from 'next/app';
+import { useCallback } from 'react'
+import dynamic from 'next/dynamic';
+import "@awsui/global-styles/index.css";
+import './App.css';
+import './index.css';
+
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { SnackbarProvider } from 'notistack';
+import { Provider } from 'react-redux';
+
+import { store } from '../store';
+
+const theme = createTheme({
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          fontSize: "14px",
+        }
+      }
+    }
+  },
+  palette: {
+    primary: {
+      main: "rgb(236, 114, 17)",
+    },
+  },
+  shape: {
+    borderRadius: "1px"
+  },
+  typography: {
+    fontSize: 20,
+    button: {
+      fontWeight: "700",
+      textTransform: "none"
+    }
+  }
+});
+
+declare global {
+  interface Window { 
+    ace: any; 
+    editor: any; 
+  }
+}
+
+function App({ Component, pageProps }: AppProps) {
+  const onAceLoad = useCallback(() => {
+    window.editor = window.ace.edit('editor')
+    window.ace.config.set('basePath', '/third-party/ace-1.4.13/')
+    window.ace.config.set('loadWorkerFromBlob', false);
+    window.ace.config.set('showFoldWidgets', false);
+    window.ace.config.set('showPrintMargin', false);
+  }, [])
+
+
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Pcluster Manager</title>
+      </Head>
+      <Provider store={store}>
+        <ThemeProvider theme={theme}>
+          <SnackbarProvider>
+            <Component {...pageProps} />
+          </SnackbarProvider>
+        </ThemeProvider>
+      </Provider>
+      <div id="editor"></div>
+      <Script
```

Comment:
thought(non-blocking): it would have been better to directly import this thing inside the bundle directly, but we'll dive into this issue another time